src/Entity/MajorVersion.php

```diff
@@ -194,29 +183,29 @@ public function getDescription(): string
         return $this->description;
     }
 
-    public function setMaintainedUntil(?\DateTimeImmutable $maintainedUntil): void
+    public function setMaintainedUntil(?\DateTimeImmutable $dateTimeImmutable): void
     {
-        $this->maintainedUntil = $maintainedUntil;
+        $this->maintainedUntil = $dateTimeImmutable;
```

Comment:
Disagree with this change.  It looks like just an auto-replace gone awry.  This is a value object, not a service, so naming it for its type is not correct.

Reply:
Agree, I don't like it too